spec/requests/api/connect/v3/repositories/installer_controller_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Api::Connect::V3::Repositories::InstallerController, type: :request do
+  include_context 'version header', 3
+
+  let(:url) { '/connect/repositories/installer' }
```

Comment:
Request specs let us use route helpers, which I think is a good idea:
`connect_repositories_installer_path`